### ISSUE TITLE
Astonishingly large icons

### ISSUE THREAD

**Issue by PPokorski** [duckietown/hatchery#14]
Hi guys!
Thanks a lot for fixing the last issue I posted. As I mentioned, I really like this plugin! Recently, however, I found some... inconvenience while using it. Namely, this: 
![Some problem](https://i.ibb.co/Hrk32k0/Screenshot-from-2019-04-25-00-19-00.png)
# Environment
OS: Ubuntu 16.04.6 LTS
CLion: 2018.2.5
Hatchery: 0.3.2

Thanks a lot for your work! You're great!

**Edit:**
In CLion 2019.1.2 the problem is no more, so you can do with this issue whatever you want :)

**Comment by breandan** [comment]
Good catch, it appears we are using SVG icons which are [only supported](https://www.jetbrains.org/intellij/sdk/docs/reference_guide/work_with_icons_and_images.html#image-formats) for `2018.2+`, so it will create issues on older platform versions. I think Hatchery will target `2019+` going forward due to [recent improvements](https://blog.jetbrains.com/clion/2019/03/clion-2019-1-embedded-dev-clangformat-memory-view/) in the CLion IDE that are helpful for ROS development. Thank you for reporting this issue!

**Comment by PPokorski** [comment]
Actually it was an accident I stumbled upon it. I had old version of CLion on my laptop :P
Thank you for answer!

**Comment by asimonov-hitachi** [comment]
I see the same issue with pycharm 2018.2.5 on ubuntu18
probably time to upgrade pycharm :)
